istore--0.1.1--0.1.2.sql

```diff
@@ -1,38 +1,107 @@
 ----functions----
+CREATE FUNCTION istore_agg_finalfn_pairs(internal)
+ RETURNS istore
+ LANGUAGE c
+ IMMUTABLE STRICT
+AS 'istore', $function$istore_agg_finalfn_pairs$function$;
+
+CREATE OR REPLACE FUNCTION bigistore_agg_finalfn(internal)
+ RETURNS bigistore
+ LANGUAGE c
+ IMMUTABLE STRICT
+AS 'istore', $function$bigistore_agg_finalfn_pairs$function$;
+
+CREATE FUNCTION istore_max_transfn(internal, bigistore)
+ RETURNS internal
+ LANGUAGE c
+ IMMUTABLE
+AS 'istore', $function$bigistore_max_transfn$function$;
+
+----
+CREATE FUNCTION istore_min_transfn(internal, bigistore)
+ RETURNS internal
+ LANGUAGE c
+ IMMUTABLE
```

Comment:
to put this into reference a bit:
> If the state transition function is declared "strict", then it cannot be called with null inputs. With such a transition function, aggregate execution behaves as follows. Rows with any null input values are ignored (the function is not called and the previous state value is retained). If the initial state value is null, then at the first row with all-nonnull input values, the first argument value replaces the state value, and the transition function is invoked at each subsequent row with all-nonnull input values. This is handy for implementing aggregates like max. Note that this behavior is only available when state_data_type is the same as the first arg_data_type. When these types are different, you must supply a nonnull initial condition or use a nonstrict transition function.
(from https://www.postgresql.org/docs/9.4/static/sql-createaggregate.html)